### DISCUSSION TITLE
24/11/11 WAVE 1(구축) KPT 회고

### DISCUSSION THREAD

**Discussion by laigasus** [cloud2scape/.github#37] [Announcements]
### Keep
좋았던 점, 앞으로도 계속되었으면 하는 점

### Problem
문제점, 아쉬웠던 점

### Try
앞으로의 일정 계획 등 keep, problem 종합해서 기술

### 이번주

**Comment by laigasus** [comment]
### Keep
- 팀 관계 
- 명확한 role

### Problem
- 포팅 일정 지연
- Discussion 사용 미숙

### Try
- 사이좋게 지내요

### 이번 주
- [x] EKS 포팅
- [x] Springboot 로그 설계
- [x] 전사 지원
- [x] 서킷 브레이커 구현(2023.0.3 프레임워크 문제로 mvc router 에서 구현 불가)
- [ ] Replication DB 연결(+CQRS 패턴)
- [x] Feign 걷어내기(문제 해결함)
- [x] k8s 워크로드 리소스 설계 구체화

**Comment by gimsoomin** [comment]
### Keep
- 테라폼, 새로운 세계

### Problem
- 진도가 느림
- GUI로 구성할걸

### Try
- redis, rds gui 구성
- 테라폼 자동화 학습

### 이번 주
- [ ] RDS 단일 구성
- [ ] Redis 단일 구성
- [ ] Argo CD 구축

**Comment by pcj1792** [comment]
### Keep
- 의사소통

### Problem
- loki

### Try
- loki-distributed
- 대쉬보드 생성

### 이번 주
- [x] loki 연동 후 log 테스트
- [x] loki에서 파일 테일러링이 가능한지, kafka 혹은 kinesis 쪽으로 메세지를 produce 기능이 있는지 확인해볼 것
- [x] 애플리케이션에서 필요한 모니터링 항목 전달하기  -- wiki에 작성
- [ ] 필요 항목들로 구성된 grafana dashboard 만들기

**Comment by ssedogdoom** [comment]
### Keep
- 인프라 구축 
- 의사소통 

### Problem
- eks porting 문제
- eks랑 rds 연결 문제 : rds를 연결해야 파드가 안죽음
- 인프라 전체 연결 조정 필요

### Try
- 인프라 전체 연결 구축

### 이번 주
- [x] 인프라 전체 연결 구축 (eks 내부 포팅 제외)
